values.yaml

```diff
@@ -67,7 +67,11 @@ st2:
   config: |
     [api]
     allow_origin = '*'
-
+  
+  #Override Definitions can be added here. 
+  # https://docs.stackstorm.com/latest/packs.html#overriding-pack-defaults
```

Comment:
Similar to `st2.packs`, it would be also nice to provide a commented example in the Helm values for config overrides so users could configure it more easily.

Reply:
I can get this added! Not a problem at all (:

Reply:
Added the commented example.